t_custom_branges/test_http_uri_brange.py

```diff
@@ -0,0 +1,95 @@
+"""Functional tests for custom uri brange."""
+
+__author__ = "Tempesta Technologies, Inc."
+__copyright__ = "Copyright (C) 2024 Tempesta Technologies, Inc."
+__license__ = "GPL2"
+
+from framework import tester
+from framework.parameterize import param, parameterize, parameterize_class
+
+DEPROXY_CLIENT = {
+    "id": "deproxy",
+    "type": "deproxy",
+    "addr": "${tempesta_ip}",
+    "port": "80",
+}
+
+DEPROXY_CLIENT_H2 = {
+    "id": "deproxy",
+    "type": "deproxy_h2",
+    "addr": "${tempesta_ip}",
+    "port": "443",
+    "ssl": True,
+}
+
+
+class HttpUriBrangeBase(tester.TempestaTest):
+    backends = [
+        {
+            "id": "server-1",
+            "type": "deproxy",
+            "port": "8000",
+            "response": "static",
+            "response_content": (
+                "HTTP/1.1 200 OK\r\n"
+                + "Date: test\r\n"
+                + "Server: deproxy\r\n"
+                + "Content-Length: 0\r\n\r\n"
+            ),
+        },
+    ]
+
+    tempesta_tmpl = """
+		server ${server_ip}:8000;
+
+		listen 80;
+		listen 443 proto=h2;
+
+		tls_certificate ${tempesta_workdir}/tempesta.crt;
+		tls_certificate_key ${tempesta_workdir}/tempesta.key;
+		tls_match_any_server_name;
+
+		http_uri_brange %s;
+	"""
+
+
+@parameterize_class(
+    [
+        {
+            "name": "Http",
+            "clients": [DEPROXY_CLIENT],
+        },
+        {
+            "name": "H2",
+            "clients": [DEPROXY_CLIENT_H2],
+        },
+    ]
+)
+class UriBrange(HttpUriBrangeBase):
```

Comment:
```suggestion
class TestUriBrange(HttpUriBrangeBase):
```
Please use `Test` prefix for test class. I will add this to README in tempesta-test repo.